### ISSUE TITLE
[v2.13.2] contract_create_transaction.go:233:7: body.FileID undefined

### ISSUE THREAD

**Issue by tomzhenghedera** [hashgraph/hedera-sdk-go#488]
### Description

While compiling a Smart contract code with v2.13.2, got this compile error

```
# github.com/hashgraph/hedera-sdk-go/v2
........pkg/mod/github.com/hashgraph/hedera-sdk-go/v2@v2.13.2/contract_create_transaction.go:233:7: body.FileID undefined (type *services.ContractCreateTransactionBody has no field or method FileID)
........pkg/mod/github.com/hashgraph/hedera-sdk-go/v2@v2.13.2/contract_create_transaction.go:281:7: body.FileID undefined (type *services.ContractCreateTransactionBody has no field or method FileID)
```

I tried to clean up all the build cache and pkg contents in the env, and didn't make any diff.  This feels like something to do with protobuf.

### Steps to reproduce

Any code that does something like this, as this happens at compile time.

```
    // Instantiate the contract instance
    contractTransactionID, err := hedera.NewContractCreateTransaction().
        SetNodeAccountIDs([]hedera.AccountID{node}).
        SetMaxTransactionFee(hedera.NewHbar(feeMax)).
        SetGas(gasCapMax).
        //SetConstructorParameters(params).
        SetBytecodeFileID(fid).
        SetAdminKey(client.GetOperatorPublicKey()).
        Execute(client)
```

### Hedera network

other

### Version

v2.13.2

### Operating system

Linux

**Comment by andrix10** [comment]
Yep looks like you are somehow using the most updated protobuf from the beta, i ran some tests on both testnet and previewnet on my end and they both worked. Make sure its `github.com/hashgraph/hedera-protobufs-go v0.2.1-0.20220407062851-1cb287936c27` in go.mod.

**Comment by tomzhenghedera** [comment]
**Update**

- was able to have a clean compile with `v2.14.0-beta.1`(start everything on a clean slate)
- had the same compile error with `v2.13.1`
- hit different error with `v2.13.0`
```
# github.com/hashgraph/hedera-sdk-go/v2
......pkg/mod/github.com/hashgraph/hedera-sdk-go/v2@v2.13.0/hbar_allowance.go:59:44: undefined: services.GrantedCryptoAllowance
......pkg/mod/github.com/hashgraph/hedera-sdk-go/v2@v2.13.0/hbar_allowance.go:71:41: undefined: services.CryptoWipeAllowance
...
...
...
```

**Comment by andrix10** [comment]
Yes it looks like something is going with your go.mod/go.sum, seems like the protobufs are getting mixed up for you.

**Comment by tomzhenghedera** [comment]
Could be.  But I did clean up all the *.sum files and the env while compiling with v2.23.2, same way before I moved to 2.14.x.  I will try again.

**Comment by tomzhenghedera** [comment]
I tried few times and did compile with v2.13.2 clean this time.  The way I did different w v2.13.2 is to require the 

```
github.com/hashgraph/hedera-protobufs-go v0.2.1-0.20220407062851-1cb287936c27
```
explicitly.  It didn't have it before so either my env didn't get cleaned up fully or somehow the `go get` didn't get the right version loaded.

But looks like it's related to the env not the source.  Feel free to close the ticket and thanks!
